### DISCUSSION TITLE
Display Edit Menu Bar

### DISCUSSION THREAD

**Discussion by evanasc** [requarks/wiki#6980] [Help / Questions] [answered]
I have a question: when disabling the "Display Edit Menu Bar" option, shouldn't the menu in the page header be hidden from the user?

Or it would have another purpose.

Current Version: 2.5.300
![Wikijs](https://github.com/requarks/wiki/assets/87073289/11154975-cfae-4b7d-9d70-6f182789678a)

**Comment by NGPixel** [comment, marked as answer]
That's not what it's referring to. The edit menu bar is one or two buttons that are added to the page header. It's not the buttons in the top right corner of the site. These are controlled based on the user permissions.
